tests/test_other.py

```diff
@@ -8910,12 +8911,35 @@ def test_wasm_sourcemap_dead(self):
                     '--dwarfdump-output',
                     path_from_root('tests', 'other', 'wasm_sourcemap_dead', 't.wasm.dump'),
                     '-o', 'a.out.wasm.map',
-                    path_from_root('tests', 'other', 'wasm_sourcemap_dead', 't.wasm')]
+                    path_from_root('tests', 'other', 'wasm_sourcemap_dead', 't.wasm'),
+                    '--basepath=' + os.getcwd()]
     run_process(wasm_map_cmd, stdout=PIPE, stderr=PIPE)
     output = open('a.out.wasm.map').read()
     # has only two entries
     self.assertRegexpMatches(output, r'"mappings":\s*"[A-Za-z0-9+/]+,[A-Za-z0-9+/]+"')
 
+  @no_fastcomp()
+  def test_wasm_sourcemap_relative_paths(self):
+    def test(infile, source_map_added_dir=''):
+      expected_source_map_path = os.path.join(source_map_added_dir, 'a.cpp')
+      print(infile, expected_source_map_path)
```

Comment:
Are these leftover prints or necessary in tests?

Reply:
They aren't necessary, but they make following the output easier when a test fails on CI, so a bunch of tests have them.